docusaurus/docs/getting-started.md

```diff
@@ -6,14 +6,14 @@ sidebar_label: Getting Started
 
 ### Installation a specific hook like useDidMount
 
-    npm i -s @rooks/use-did-mount
+    npm i -s @rooks/useDidMount
```

Comment:
So we don't have installing a specific hook any more. so just remove this paragraph and name the other paragraph as `Installation` instead of `### Installation all the hooks in a single package`